dll/win32/browseui/shellbars/CBandSiteMenu.cpp

```diff
@@ -155,6 +142,31 @@ HRESULT CBandSiteMenu::_CreateBuiltInISFBand(UINT uID, REFIID riid, void** ppv)
     LPITEMIDLIST pidl;
     HRESULT hr;
 
+    switch (uID)
+    {
+        case IDM_TASKBAR_TOOLBARS_DESKTOP:
+            if (m_DesktopPidl != NULL)
```

Comment:
Maybe putting the contents of the "case xxx:" inside curly brackets would be nice (allows also to fold in code editors).

Reply:
Done!